src/coreclr/jit/emitfmtswasm.h

```diff
@@ -28,6 +28,8 @@ enum ID_OPS
 
 IF_DEF(NONE,    IS_NONE, NONE)
 IF_DEF(OPCODE,  IS_NONE, NONE) // <opcode>
+IF_DEF(TYPE,    IS_NONE, NONE) // <opcode> <type 0x40>
```

Comment:
Usually in the context of WASM instruction encodings, `type` operands are relocatable ULEB128s (no need for a new instruction format). But the block types are indeed a special case.
```suggestion
IF_DEF(BLOCK,   IS_NONE, NONE) // <opcode> <0x40>
```